apps/desktop/electron/src/service/sources.ts

```diff
@@ -84,12 +85,14 @@ export function getSourceFolders(sourceId: string, query?: string): Observable<S
 function downloadFileOrLink(
   sourceId: string,
   item: SyncItem,
-): Observable<{ type: 'blob' | 'link'; blob?: Blob; link?: any }> {
+): Observable<{ type: 'blob' | 'link' | 'text'; blob?: Blob; link?: any; text?: TextField }> {
```

Comment:
here where `TextField` dependency is coming from `@nuclia/core` and not `SDK-core`. I'm not sure which one is the good one but I feel like we should be consistent, no?